tools/codegen/gentable.py

```diff
@@ -192,7 +200,7 @@ def generate(self, path, template="default"):
             for option in column.options:
                 # Only allow explicitly-defined options.
                 if option in COLUMN_OPTIONS:
-                    column_options.append(COLUMN_OPTIONS[option])
+                    column_options.append("ColumnOptions::" + COLUMN_OPTIONS[option])
```

Comment:
We could insert this `ColumnOptions::` prefix in the template, but there's precent for adding prefixes within the python code already. :/